app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,18 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
+  process resize_to_fit: [360, 360]
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def extension_white_list
+    %w(jpg jpeg gif png)
+  end
+
+  def default_url(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `default_url(*)` if you want the method to accept any arguments but don't care about them.